openrtb_ext/imp_adoppler.go

```diff
@@ -1,5 +1,6 @@
 package openrtb_ext
 
 type ExtImpAdoppler struct {
+	Client string `json:"client"`
```

Comment:
Please open a PR on the docs repository to add the new client parameter to this doc: https://github.com/prebid/prebid.github.io/blob/master/dev-docs/bidders/adoppler.md

Reply:
Openned: https://github.com/prebid/prebid.github.io/pull/2422